### ISSUE TITLE
Open Ulmus Course Build

### ISSUE THREAD

**Issue by mmilutinovic013** [elmsln/openulmus.org#7]
We need an intro to open ulmus course for a second Proof of Concept course. This will include information on how open ulmus works, how to set up a new course, and how to become involved with the community.

**Comment by gusaus** [comment]
@mmilutinovic1313 Keeping in mind that we're prioritizing projects on available (or potential) budget  I think this could be prioritized over https://github.com/elmsln/openulmus.org/issues/6  

Main reason being we could include the cost of producing such a class (including docs, twitch sessions, vids, etc.) in the overall scope of the Open Ulmus project  https://github.com/elmsln/openulmus.org/issues/1 

As Open Ulmus runs on ELMS, it seems like we could also incorporate material planned for the Intro to ELMS class.

**Comment by mmilutinovic013** [comment]
Let's bring this into scope then. I will create an issue tag for the creation of this course build and how it relates to the drupal open learning relaunch.

**Comment by mmilutinovic013** [comment]
@btopro is it possible to make me a pseudo-mod so I can add labels to issues in this space?

**Comment by btopro** [comment]
done

**Comment by mmilutinovic013** [comment]
Did we establish that we are merging this with the Drupal Open Learning course? Or am I losing my mind? 😛

**Comment by mmilutinovic013** [comment]
@kat-wehr I'm tagging you in this discussion as I will be scoping out the initial course build here! More info to come.

**Comment by cleverington** [comment]
@mmilutinovic1313 In reply to comment #7#issuecomment-339192201:

Seems like it'll be a course directly linked to OpenUlmus, as it will be used/referenced by *all* Open Learning groups, not just Drupal.

Mentally I see it as ( @gusaus / @mmilutinovic1313 / @btopro , definitely review here... I'll update the draw.io diagram to reflect any changes to the overall infrastructure, if needed.)

## Open Learning
Parent/Controller for OpenUlmus (no platform, no content. _MIGHT_ actually be simpler to just have it as an extension of OpenUlmus long-term, but easier in the here-and-now to have them completely separated)
### OpenUlmus
* Global to all Open Learning entities (Drupal, maybe Ruby, maybe React, etc.)
* Training space on OpenUlmus.
* Has own Courses
* Has own Trainings

**Note** - OpenUlmus is both the 'Server' (digitally) and the upper-most parent entity (infrastructurally) to the OpenUlmus platform. Thus, OpenUlmus *is* the platform *and* OpenUlmus has a community space on the platform.
#### Drupal Open Learning
* Entity for Drupal groups and OpenUlmus sponsored/owned _Drupal_ based course materials
* Training space on OpenUlmus.
* Has own Courses (possibly sponsored by the OpenUlmus Team)
* Has own Trainings (possibly sponsored by the OpenUlmus Team)
##### Drupal Dojo
* Child Entity of Drupal Open Learning.
* Training space on OpenUlmus.
* Has own Courses (mentored by the Drupal Open Learning Team)
* Has own Trainings (mentored by the Drupal Open Learning Team)
##### Austin Drupal Dojo
* On paper, is child of Drupal Dojo.
* In reality, child of Drupal Open Learning.
* Training space on OpenUlmus.
* Has own Courses (mentored by the Drupal Open Learning Team)
* Has own Trainings (mentored by the Drupal Open Learning Team)
#### React Open Learning
* Entity for React groups and OpenUlmus sponsored/owned _React_ based course materials
* Training space on OpenUlmus.
* Has own Courses (mentored by the OpenUlmus Team)
* Has own Trainings (mentored by the OpenUlmus Team)
##### React Dojo
* Child Entity of React Open Learning.
* Training space on OpenUlmus.
* Has own Courses (mentored by the React Open Learning Team)
* Has own Trainings (mentored by the React Open Learning Team)
#### WordPress Open Learning
* Entity for WordPress groups and OpenUlmus sponsored/owned _WordPress_ based course materials
* Training space on OpenUlmus.
* Has own Courses (mentored by the OpenUlmus Team)
* Has own Trainings (mentored by the OpenUlmus Team)
##### WordPress Dojo
* Child Entity of WordPress Open Learning.
* Training space on OpenUlmus.
* Has own Courses (mentored by the WordPress Open Learning Team)
* Has own Trainings (mentored by the WordPress Open Learning Team)

**Comment by mmilutinovic013** [comment]
When you say 'instance' do you mean its own network of courses? Or is it an individual Open Ulmus platform? I believe it's the first one but I wanted to clarify?

**Comment by cleverington** [comment]
Instance is probably bad verbiage on my part here.  Probably better to say:

* Community space on OpenUlmus

(Note updated.)

**Comment by btopro** [comment]
yeah, Instance to me sounds like Deployment of the architecture. Maybe `Training space` or something like that. ELMS:LN's connotation of a course would equate to 1 training in my mind. People buy access to advanced trainings. Those trainings are all part of their organization / tagged appropriately.

I think another inflection point here would be that everything is publicly listed. As in there aren't like "Oh your a grand master ninja then you get access to..." sorta stuff. You pay for advanced trainings, you unlock it, community gets paid, but everyone is aware of all the possible trainings out there.

**Comment by btopro** [comment]
also worth noting -- Orgs like Drupal / WP that are massive could potentially need to be split off into their own deployments of the architecture just thinking from a scale / branding perspective. I could see a road towards like white-labing open ulmus for bigger communities but that's a back of your mind sorta thought.

**Comment by cleverington** [comment]
*nods*  That's why I added the note to add a layer separation to the "Open Ulmus Platform" and "Open Ulmus" entity.  Perhaps long-term we'll end up just referring to the platform simply as ELMS or something, so (from an EndUser standpoint) it doesn't matter if we're using one instance of the architecture or fifteen, it all has the same 'base'.

Will replace `Community space on OpenUlmus` with `Training space on OpenUlmus` in the note.

Will also update the draw.io diagram later this afternoon for a more accurate representation of the platform to have to parts: digital and infrastructure.   One side will show a digital representation of the architecture and the other will show the working/business representation of the architecture.  (Both as indicated in above comment. eg: Child of this, but still just a training space on OpenUlmus)
